docs/conf.py

```diff
@@ -224,7 +227,12 @@
 #latex_preamble = ''
 
 # Documents to append as an appendix to all manuals.
-#latex_appendices = []
+latex_appendices = [
+    'support',
+    'resources',
+    'changes',
+    'authors',
+]
```

Comment:
so, does this mean the appendices do not show up on the website (html output)?

man page?